app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java

```diff
@@ -1676,6 +1676,11 @@ public void onAction(String action) {
                 if (aSelection.isActionAvailable(action)) {
                     aSelection.execute(action);
                 }
+                if (GeckoSession.SelectionActionDelegate.ACTION_COPY.equals(action) &&
+                        aSelection.isActionAvailable(GeckoSession.SelectionActionDelegate.ACTION_UNSELECT)) {
+                    // Don't keep the text selected after it's copied.
+                    aSelection.execute(GeckoSession.SelectionActionDelegate.ACTION_UNSELECT);
```

Comment:
One thing we can do in a follow up is use the shorthand calls, so You can just call `aSelection.unselect()` instead of calling the more verbose `execute()`. The fact that you have to check if the action is available sort of sucks though.